powerhome/playbook
# [NUXE-260] SAMPLE PAGE - Image aspect ratio on News View

#### Screens
**Rails**
![Screen Shot 2020-11-18 at 10 15 54 AM](https://user-images.githubusercontent.com/64423490/99549022-3d422180-2987-11eb-8a7e-fc4c1a4321c8.png)

**React**
<img width="1779" alt="Screen Shot 2020-11-18 at 10 28 52 AM" src="https://user-images.githubusercontent.com/64423775/99550649-f05f4a80-2988-11eb-9a11-38974c19cdad.png">



#### Breaking Changes

No, we are fixing issues on the sample news magazine with Javascript.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-260

#### How to test this

Go into samples and select the news magazine example. Make sure to see the page loading with images and that the images fit the card body.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
